### PR TITLE
ROX-17144: Add integration tests for Workload CVE deploy page

### PR DESCRIPTION
**Pull request by dvail**
## Description

As titled.

Tests focus on items that:
- integrate multiple components on the page
- cannot easily be tested with component tests alone
- are independent from the data, or lack thereof, returned via the API

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

Local Cypress run

CI

### CODE REVIEW

**Comment by dvail** [comment]
Current dependencies on/for this PR:
* `master`
  * **PR #9055** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/9055?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/9055?utm_source=stack-comment).

File: ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/SummaryCards/BySeveritySummaryCard.tsx [hunk 1]
```diff
@@ -73,7 +73,12 @@ function BySeveritySummaryCard({
                                     spaceItems={{ default: 'spaceItemsSm' }}
                                     alignItems={{ default: 'alignItemsCenter' }}
                                 >
-                                    {Icon && <Icon color={hasNoResults ? textColor : undefined} />}
+                                    {Icon && (
```
Comment (dvail) [inline ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/SummaryCards/BySeveritySummaryCard.tsx:76]: Adds a semantic title to the icons so:
1. We can more easily grab onto this element, siblings via Cypress
2. There is an accessible name for an icon that carries meaning more than just being a decorative element

File: ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/deploymentSingle.test.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            cy.visit(`${href}?page=2`);
+            cy.get(selectors.resourcesTab).click();
+            cy.location('search').should('not.contain', 'page=');
+        });
+
+        cy.location('href').then((href) => {
+            cy.visit(`${href}?page=2`);
+            cy.get(selectors.vulnerabilitiesTab).click();
+            cy.location('search').should('not.contain', 'page=');
+        });
```
Comment (sachaudh) [inline ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/deploymentSingle.test.js:95]: Do you think it's worth separating the test into two different ones?
1. Check to see if the tab content shows the correct UI elements (table with specific columns) for the tab
2. Check for the correct pagination state between changing tab views

**Review by sachaudh — APPROVED** [review]
LGTM 🚢

**Pull request merged**